index.js

```diff
@@ -60,6 +63,45 @@ function loadFile (prefix, { customSelection, factory }) {
   })
 }
 
+function expand (prefixed, types = []) {
+  const [prefix, term] = prefixed.split(':')
+  if (!prefix || !term) {
+    return ''
+  }
+
+  const baseIRI = allPrefixes[prefix]
+  if (!baseIRI) {
+    throw new Error(`Unavailable prefix '${prefix}:'`)
+  }
+
+  const iri = `${baseIRI}${term}`
+  if (!types.length) {
+    return iri
+  }
+
+  return Promise.resolve()
```

Comment:
How about extracting this to an `async function` to get rid of the promises?

```suggestion
  return expandWithCheck(prefixed, types)
```

Reply:
Good idea!